ydkgen/printer/doc/doc_printer.py

```diff
@@ -27,8 +27,9 @@
 
 
 class DocPrinter(object):
-    def __init__(self, ctx):
+    def __init__(self, ctx, language='py'):
```

Comment:
Suggest not providing default value of 'py' for language. Generally, having arguments with default values is not a clean code programming practice and it increases chances of bugs. We may need to clean up the ydkgen/sdk code wherever we have default value aguments